### PR TITLE
[MSYS-812]: Enable expeditor

### PR DESCRIPTION
**Pull request by NAshwini**
Signed-off-by: NAshwini <ashwini.nehate@msystechnologies.com>

### CODE REVIEW

**Comment by NAshwini** [comment]
@btm , @Vasu1105 , Please review PR so I can test config files by `expenditor` commands.

File: .expeditor/config.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  notify_channel: sustaining-notify
+
+rubygems:
+  - knife-ec2
+
+github:
+  # The tag format to use (e.g. v1.0.0)
+  version_tag_format: "v{{version}}"
+  # The Github Team primarily responsible for handling incoming Pull Requests.
+  maintainer_group: chef/msys-developers
```
Comment (tduffield) [inline .expeditor/config.yml]: This value has been deprecated. Please remove.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ Note: this log contains only changes from knife-ec2 release 0.8.0 and later
 -- it does not contain the changes from prior releases. To view change history
 prior to release 0.8.0, please visit the [source repository](https://github.com/chef/knife-ec2/commits).
 
+<!-- latest_release -->
+<!-- latest_release -->
+
+<!-- release_rollup -->
+<!-- release_rollup -->
+
+<!-- latest_stable_release -->
```
Comment (tduffield) [inline CHANGELOG.md]: You need an ending `<!-- latest_stable_release -->`

File: .expeditor/update_version.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# After a PR merge, Chef Expeditor will bump the PATCH version in the VERSION file.
+# It then executes this file to update any other files/components with that new version.
+#
+
+set -evx
+
+sed -i -r "s/^(\s*)VERSION = \".+\"/\1VERSION = \"$(cat VERSION)\"/" /lib/knife-ec2/version.rb
```
Comment (tduffield) [inline .expeditor/update_version.sh]: make this path relative not absolute: `lib/knife-ec2/version.rb`

File: .expeditor/config.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        - "Expeditor: Skip All"
+  - bash:.expeditor/update_version.sh:
+      only_if: built_in:bump_version
+  - built_in:update_changelog:
+      ignore_labels:
+        - "Meta: Exclude From Changelog"
+        - "Expeditor: Exclude From Changelog"
+        - "Expeditor: Skip All"
+  - built_in:build_gem:
+only_if: built_in:bump_version
```
Comment (tduffield) [inline .expeditor/config.yml]: please make sure this is indented properly

File: .expeditor/config.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
+# Documentation available at http://expeditor-docs.es.chef.io/
```
Comment (tduffield) [inline .expeditor/config.yml]: docs are now at http://expeditor.es.chef.io

**Comment by NAshwini** [comment]
Thank you @tduffield and @rhass  I have updated the files as per review comments.

**Comment by tduffield** [comment]
If you manually modify the `VERSION` file, the `built_in:bump_version` step will assume that you have manually set the version to be what you want it to be, and will no-op. This can cause a problem if the version (in this case `0.18.0` already exists). So you have two options: 

1. Manually change the version to something like `0.19.0`
2. Create and apply one of the skip labels (e.g. `Expeditor: Skip All`) to ensure that no Expeditor actions take place

**Comment by NAshwini** [comment]
Ok @tduffield, after merging this PR will go with 2nd option of the skip labels (e.g. `Expeditor: Skip All`).

Thank you.

**Comment by tduffield** [comment]
@NAshwini you'll need to apply the label to this PR, otherwise you'll run into an error.

**Pull request merged**